packages/core/src/Serialization/AuxiliaryData/AuxiliaryData.ts

```diff
@@ -228,10 +228,13 @@ export class AuxiliaryData {
    */
   toCore(): Cardano.AuxiliaryData {
     const scripts = this.#getCoreScripts();
-    return {
-      blob: this.#metadata ? this.#metadata.toCore() : undefined,
-      scripts: scripts.length > 0 ? scripts : undefined
+    const auxiliaryData: Cardano.AuxiliaryData = {
+      blob: this.#metadata ? this.#metadata.toCore() : undefined
```

Comment:
shouldn't blob also be omitted when undefined?
```diff
```

Reply:
@mirceahasegan `blob: undefined` is the expected behavior compared to the existing Providers.